simtools/corsika/corsika_output.py

```diff
@@ -0,0 +1,1301 @@
+        return np.array(hist_1D_list), np.array(x_edges_list)
+
+    def get_photon_radial_distr(self, bins=None, max_dist=None, density=True):
+        """
+        Get the radial distribution of the photons on the ground in relation to the center of the
+        array.
+
+        Parameters
+        ----------
+        bins: float
+            Number of bins of the radial distribution.
+        max_dist: float
+            Maximum distance to consider in the 1D histogram (in meters).
+        density: bool
+            If True, returns the density distribution. If False, returns the distribution of counts.
+
+        Returns
+        -------
+        np.array
+            The counts of the 1D histogram with size = int(max_dist/bin_size).
+        np.array
+            The edges of the 1D histogram in meters with size = int(max_dist/bin_size) + 1.
+        """
+
+        if max_dist is None:
+            max_dist = np.amax(
+                [
+                    self.hist_config["hist_position"]["x axis"]["start"].to(u.m).value,
+                    self.hist_config["hist_position"]["x axis"]["stop"].to(u.m).value,
+                    self.hist_config["hist_position"]["y axis"]["start"].to(u.m).value,
+                    self.hist_config["hist_position"]["y axis"]["stop"].to(u.m).value,
+                ]
+            )
+        if bins is None:
+            bins = (
+                np.amax(
+                    [
+                        self.hist_config["hist_position"]["x axis"]["bins"],
+                        self.hist_config["hist_position"]["y axis"]["bins"],
+                    ]
+                )
+                // 2
+            )  # //2 because of the 2D array going into the negative and
+            # positive axis
+        edges_1D_list, hist1D_list = [], []
+
+        hist2D_values_list, x_position_list, y_position_list = self.get_2D_photon_position_distr(
+            density=density
+        )
+
+        for i_hist, _ in enumerate(x_position_list):
+            hist1D, edges_1D = convert_2D_to_radial_distr(
+                hist2D_values_list[i_hist],
+                x_position_list[i_hist],
+                y_position_list[i_hist],
+                bins=bins,
+                max_dist=max_dist,
+            )
+            edges_1D_list.append(edges_1D)
+            hist1D_list.append(hist1D)
+        return np.array(hist1D_list), np.array(edges_1D_list)
+
+    def get_photon_wavelength_distr(self):
+        """
+        Get histograms with the wavelengths of the photon bunches.
+
+        Returns
+        -------
+        np.array
+            The counts of the wavelength histogram.
+        np.array
+            The edges of the wavelength histogram in nanometers.
+
+        """
+        return self._get_hist_1D_projection("wavelength")
+
+    def get_photon_time_distr(self):
+        """
+        Get the distribution of the emitted time of the Cherenkov photons. The clock starts when the
```

Comment:
Should that be `time of emission`? Otherwise it sounds like C photons emit times.